colabs/Pandas_Profiling .ipynb

```diff
@@ -0,0 +1,10911 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
```

Comment:
Kod
```


sample_title_basics = sample_title_basics.toPandas()
profile_basics = ProfileReport(sample_title_basics, title="Pandas Profiling Report - title basics", minimal=True)
profile_basics.to_file("title_basics.html")
```

Wygląda na mocno replikowalny. Możemy go  opakować we funkcję?

Reply:
Wrzuciłam w funkcję profile()

Reply:
```
def profile(data, sample_size, report_name):
  sample = data.sample(sample_size)
  data_pd = sample.toPandas()
  profile = ProfileReport(data_pd, title = f"Pandas Profiling Report {report_name}")
  profile.to_file(f"reports/ {report_name}.html")
```
Są tu pewne zarzuty, ale je mogę puścić na tym etapie. Ale spacje w nazwie to bym usunął.

Reply:
poprawione